### ISSUE TITLE
ESLint issue (`root:true` is needed)

### ISSUE THREAD

**Issue by probil** [kuitos/axios-extensions#17]
First of all, thank you for the library :+1: 

But I got an issue while using this library in CI today:
```
[lint-ci] Cannot find module '/app/node_modules/axios-extensions/vuejsapp/build/webpack.base.conf.js'
[lint-ci] Error: Cannot find module '/app/node_modules/axios-extensions/vuejsapp/build/webpack.base.conf.js'
[lint-ci]     at Function.Module._resolveFilename (module.js:547:15)
[lint-ci]     at Function.Module._load (module.js:474:25)
[lint-ci]     at Module.require (module.js:596:17)
[lint-ci]     at require (internal/module.js:11:18)
[lint-ci]     at Object.exports.resolve (/app/node_modules/eslint-import-resolver-webpack/index.js:68:25)
[lint-ci]     at v2 (/app/node_modules/eslint-module-utils/resolve.js:94:23)
[lint-ci]     at withResolver (/app/node_modules/eslint-module-utils/resolve.js:99:16)
[lint-ci]     at fullResolve (/app/node_modules/eslint-module-utils/resolve.js:116:22)
[lint-ci]     at Function.relative (/app/node_modules/eslint-module-utils/resolve.js:61:10)
[lint-ci]     at remotePath (/app/node_modules/eslint-plugin-import/lib/ExportMap.js:379:30)
[lint-ci] npm ERR! code ELIFECYCLE
[lint-ci] npm ERR! errno 1
```
After a quick investigation,  I've realized that ESLint on CI is trying to extend your config with ESLint config in my project (even though `node_modules` is ignored). I think **it can be fixed by marking your ESLint config as roo**t, so if your library is placed inside a dir it will not automatically extend the rules defined in the parent directory.
```js
"eslintConfig": {
    "root": true,    // <-- ADD IT HERE PLEASE
    "extends": "shuyun",
    "rules": {
      "object-curly-spacing": "off",
      "no-process-env": "off"
    }
  },
```
> By default, ESLint will look for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place `"root": true` inside the `eslintConfig` field of the `package.json` file or in the `.eslintrc.*` file at your project’s root level. ESLint will stop looking in parent folders once it finds a configuration with `"root": true`.

From: https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

**Comment by probil** [comment]
Temporary fixed it by changing local ESLint config this way:
```js
"settings": {
  "import/ignore": ["node_modules"]
},
```
Maybe this will help someone spend less time on that

**Comment by kuitos** [comment]
Thanks for your information! I will add it soon.

**Comment by kuitos** [comment]
released v2.0.3
